menu/kielux_lpd_2024.json

```diff
@@ -0,0 +1,28 @@
+{
+	"version": 2024081901,
+	"title": "Kielux Linux Presentation Day",
```

Comment:
Shouldn't there be a year in here BTW? Hm I see it was missing last 2 years as well but present before. While the menu does show the date in subscript I think it's still better to include a year here too..

Reply:
It would have been okay either way, but now I added a year. Thanks for reviewing, @Wilm0r !